src/falco/chart/templates/rules-configmap.yaml

```
@@ -0,0 +1,35 @@
+# Copyright 2025 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: falco-extra-rules
+  namespace: falco
+data:
+  # Include stable rules
+  stable-rules.yaml: |-
+{{ .Files.Get "rules/stable-rules.yaml" | nindent 4 }}
+
+  # Include sandbox rules if sandbox ruleset is enabled in values.yaml
+  {{- if .Values.sandboxRulesEnabled }}
+  sandbox-rules.yaml: |-
+{{ .Files.Get "rules/sandbox-rules.yaml" | nindent 4 }}
+  {{- end }}
+
+  # Include incubating rules if incubating ruleset is enabled in values.yaml
+  {{- if .Values.incubationRulesEnabled }}
```

nit: maybe keep the naming the same as the upstream name (incubating vs incubation)
```suggestion
  {{- if .Values.incubatingRulesEnabled }}
```